ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -551,7 +542,7 @@ def management_interfaces(self):
             inserting ipv6 variables if self._mgmt_ipv6_enable is turned on
             """
             if self._ipv6_mgmt_ip is not None:
-                interface_settings["ipv6_enable"] = self._mgmt_ipv6_enable
+                interface_settings["ipv6_enable"] = self._ipv6_mgmt_ip is not None
```

Comment:
no need to run an extra check here. You have the `if` in the line above, so just set `True` here.

Reply:
Fixed, thanks!